project/src/main/kotlin/org/ionproject/core/programme/ProgrammeServices.kt

```diff
@@ -1,11 +1,11 @@
 package org.ionproject.core.programme
 
-import org.ionproject.core.common.model.Programme
-import org.ionproject.core.common.model.ProgrammeOffer
+import org.ionproject.core.programme.model.Programme
+import org.ionproject.core.programme.model.ProgrammeOffer
 import org.springframework.stereotype.Component
 
 @Component
-class ProgrammeServices(private val repo : ProgrammeRepo) {
+class ProgrammeServices(private val repo : ProgrammeRepoImpl) {
```

Comment:
Shouldn't the constructor receive a `ProgrammeRepo` instance instead of `ProgrammeRepoImpl` to let Spring handle the dependency injection with less restrictions?

Reply:
It should. Idk why it ended up like this, to be honest.

Reply:
Solved.